test/distributed/fsdp/test_fsdp_ignored_modules.py

```diff
@@ -166,25 +192,48 @@ def test_ignored_modules_nested(self):
         """Tests that passing a module with nested FSDP modules does not
         error and still ignores non-FSDP modules' parameters."""
         self.run_subtests(
-            {"use_orig_params": [False, True], "ignore_modules": [True, False]},
+            {
```

Comment:
Are we running `composable=True` twice because it ignores `use_orig_params=True` vs. `False`?